bikeshed/__init__.py

```diff
@@ -1248,6 +1262,7 @@ def preprocess(self):
         processHeadings(self, "all") # again
         addTOCSection(self)
         addSelfLinks(self)
+        processBiblioLinks(self) # again, as biblio links could be added by addIndexSection().
```

Comment:
Yeah, the ordering there is unimportant. Go for it.

(I still intend to some day add an explicit ordering mechanism, so the important orderings become more visible, but meh.)